package log

import "testing"

func TestName(t *testing.T) {
	Error("查询错i五")
}
